index.html

```diff
@@ -13,11 +13,38 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
 </head>
 <body>
-  <header>
-  </header>
+  <div class="header-banner">
+    <header class="header-title">
+      <h2>Code Journal</h2>
+    </header>
+  </div>
   <main>
+    <div data-view="entry-form" class="container">
+      <div class="row">
+          <form id="form" class="column-full">
+            <h1>New Entry</h1>
+            <div class="display-flex">
+              <img id="main-image" src="images/placeholder-image-square.jpg">
```

Comment:
This should be half the available width in the row, you should probably use the `column-half` class